pizzly/wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+# Fix ownership of output files
```

Comment:
Don't we have this chmod as one of the deferred functions that runs after a `dockerCall` invocation in Toil now? I think we can nix the wrappers...